docs/pages-for-subheaders/rke2-hardening-guide.md

```diff
@@ -204,145 +206,45 @@ rules:
-#### CIS v1.23 - Control  5.2.6 - Minimize the admission of containers with allowPrivilegeEscalation (Automated)
-#### CIS v1.23 - Control  5.2.7 - Minimize the admission of root containers (Automated)
-#### CIS v1.23 - Control  5.2.8 - Minimize the admission of containers with the NET_RAW capability (Automated)
-
-**Rationale**
-
-Those controls are defined to evaluate certain PodSecurityPolicy resources in the benchmark, but the PodSecurityPolicy API is removed starting in Kubernetes v1.25, therefore those controls fail in the scan.
-
-**Remediation**
-
-Pod Security Admission is configured by setting the Pod Security Admission Configuration Template (PSACT) `rancher-restricted` in the cluster to enforce the restriction on applicable namespaces.
-Due to the lack of utilizing PSA in the current version of the benchmark, the above controls will not be addressed until an updated version of the benchmark is available.
-
-</TabItem>
-</Tabs>
-
 ## Reference Hardened RKE2 Template Configuration
 
 The reference template configuration is used in Rancher to create a hardened RKE2 custom cluster. This reference does not include other required **cluster configuration** directives which will vary depending on your environment.
 
 
 <Tabs groupId="rke2-version">
-<TabItem value='v1.25 and Newer' default>
+<TabItem value="v1.25 and Newer" default>
 
 ```yaml
 apiVersion: provisioning.cattle.io/v1
 kind: Cluster
 metadata:
-  name: rke2-125-new
-  namespace: fleet-default
+  name: # Define cluster name
 spec:
   defaultPodSecurityAdmissionConfigurationTemplateName: rancher-restricted
-  kubernetesVersion: v1.25.6+rke2r1
+  kubernetesVersion: # Define RKE2 version
   rkeConfig:
-    etcd:
-      disableSnapshots: false
-      snapshotRetention: 5
-      snapshotScheduleCron: 0 */5 * * *
-    machineGlobalConfig:
-      cni: calico
-      disable-kube-proxy: false
-      etcd-expose-metrics: false
     machineSelectorConfig:
       - config:
           profile: cis-1.23
           protect-kernel-defaults: true
-    upgradeStrategy:
-      controlPlaneConcurrency: '1'
-      controlPlaneDrainOptions:
-        deleteEmptyDirData: true
-        disableEviction: false
-        enabled: false
-        force: false
-        gracePeriod: -1
-        ignoreDaemonSets: true
-        skipWaitForDeleteTimeoutSeconds: 0
-        timeout: 120
-      workerConcurrency: '1'
-      workerDrainOptions:
-        deleteEmptyDirData: true
-        disableEviction: false
-        enabled: false
-        force: false
-        gracePeriod: -1
-        ignoreDaemonSets: true
-        skipWaitForDeleteTimeoutSeconds: 0
-        timeout: 120
 ```
 </TabItem>
 
-<TabItem value='v1.22 up to 1.24'>
+<TabItem value="v1.24 and Older">
 
 ```yaml
 apiVersion: provisioning.cattle.io/v1
 kind: Cluster
 metadata:
-  name: rke2-v124
-  namespace: fleet-default
+  name: # Define cluster name
 spec:
-  defaultPodSecurityPolicyTemplateName: restricted-noroot
```

Comment:
Sorry that my original question is a bit dismissing. I think we should keep using the `restricted-noroot` instead of leaving the field empty to use RKE2's default because Racnher checks this field for RBAC-related operations, such as creating the proper role and rolebindings.

Reply:
I got your question now and it makes sense. Will update the guide.

Reply:
cool